xcube_cci/dataaccess.py

```diff
@@ -135,7 +135,20 @@ def _get_data_descriptor_from_metadata(self, data_id: str, metadata: dict) -> Da
                              dataset_info['temporal_coverage_end'].split('T')[0])
         var_infos = ds_metadata.get('variable_infos', {})
         var_descriptors = self._get_variable_descriptors(dataset_info['var_names'], var_infos)
-        coord_descriptors = self._get_variable_descriptors(dataset_info['coord_names'], var_infos)
+        coord_descriptors = self._get_variable_descriptors(dataset_info['coord_names'],
+                                                           var_infos,
+                                                           normalize_dims=False)
+        if 'time' not in coord_descriptors.keys() and 't' not in coord_descriptors.keys():
+            time_attrs = {
+                "units": "seconds since 1970-01-01T00:00:00Z",
+                "calendar": "proleptic_gregorian",
+                "standard_name": "time"
+            }
```

Comment:
This is how `time` looks after normalisation?

Reply:
This is what is set in the chunkstore.